bundles/AdminBundle/Resources/public/js/pimcore/report/custom/report.js

```diff
@@ -232,9 +234,7 @@ pimcore.report.custom.report = Class.create(pimcore.report.abstract, {
             plugins: ['pimcore.gridfilters'],
             stripeRows: true,
             trackMouseOver: true,
-            viewConfig: {
-                forceFit: false
-            },
+            forceFit: false,
```

Comment:
Are you sure that this shouldn't be a viewConfig setting?
Maybe "false" is the default anyway so this can go away entirely?

Reply:
I am quite sure, see https://docs.sencha.com/extjs/6.0.2/classic/Ext.grid.Panel.html#cfg-forceFit while for `viewConfig` of a grid panel "Any of the config options available for [Ext.view.Table](https://docs.sencha.com/extjs/6.0.2/classic/Ext.view.Table.html) can be specified here" - and `forceFit` is not listed there.

Sadly I cannot see the default value in the documentation, for this reason I kept it.

Reply:
ok, there is probably some Extjs magic going on :-)